src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -1515,4 +1515,31 @@ Do you want to change the channel? </value>
   <data name="TextRangeControl_Details" xml:space="preserve">
     <value>Details</value>
   </data>
+  <data name="UpdateDialogWindowTitle" xml:space="preserve">
+    <value>There is an update available</value>
```

Comment:
`UpdateDialogWindowTitle` appears to be unused

Reply:
This is used on line 12 of `UpdateContainedDialog`. It's still got the old name referring to a window, so I have now updated that to be more accurate